js/editor-ta.js

```diff
@@ -156,9 +171,8 @@ EditorTextArea.prototype.createLineElement = function(number, height) {
 EditorTextArea.prototype.updateDimentions = function() {
   const editorHeight = this.element_.getBoundingClientRect().height;
   this.dimentions.height = (editorHeight - this.verticalTextareaPadding_);
-  this.dimentions.width = (this.wrapper_.getBoundingClientRect().width -
-    this.horizontalTextareaPadding_);
-
+  this.dimentions.width = (this.wrapper_.getBoundingClientRect().width
```

Comment:
nit: revert this. google js style states newlines come after operators https://google.github.io/styleguide/jsguide.html#formatting-where-to-break